android/app/build.gradle

```diff
@@ -32,16 +32,16 @@ if (keystorePropertiesFile.exists()) {
 }
 
 android {
-    compileSdkVersion 32
+    compileSdkVersion 33
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'
     }
 
     defaultConfig {
         applicationId "com.linagora.android.teammail"
-        minSdkVersion 16
-        targetSdkVersion 30
+        minSdkVersion 19
+        targetSdkVersion 32
```

Comment:
Use 33. But please provide requirements when upgrade from 31 to 33

Reply:
Need to update flutter_inappwebview to 5.7.1